Core/Steps/SubSteps/ReleaseVersionAndMoveIssuesSubStep.cs

```diff
@@ -135,10 +135,25 @@ private bool ShouldMoveIssuesToNextVersion (string versionID, string nextVersion
       return false;
 
     _console.WriteLine("These are some of the issues that will be moved by releasing the version on jira:");
-    foreach (var issue in issuesToMove.Take(5))
-      _console.WriteLine($"'{issue.Key} - {issue.Fields.Summary}'");
+    PrintIssueListing(issuesToMove);
     _console.WriteLine("Do you want to move these issues to the new version and release the old one or just release the old version?");
 
     return _inputReader.ReadConfirmation();
   }
+
+  private void PrintIssueListing (IReadOnlyList<JiraToBeMovedIssue> issues, int maxLines = 5)
+  {
+    var maxLength = _console.Profile.Width;
+    foreach (var issue in issues.Take(maxLines))
+    {
+      var message = $"* '{issue.Key} - {issue.Fields.Summary}'";
```

Comment:
```suggestion
      var message = $"* {issue.Key} - {issue.Fields.Summary}";
```
here we don't need quotes (it's readable console output, not a log message). also, quotes without a trailing quote that goes `...` would look strange.